rpcs3/Emu/RSX/Program/FragmentProgramRegister.cpp

```diff
@@ -0,0 +1,198 @@
+#include "stdafx.h"
+#include "FragmentProgramRegister.h"
+
+#pragma optimize("", off)

```

Comment:
Remember to remove this before merging

Reply:
Thanks for the reminder. I would have forgotten about it lol